scripts/docker-run-restore-points-test.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+SCRIPT_DIR=$(dirname $0)
```

Comment:
The beginning of this script seems to be copy-pasting things from other scripts. I guess there's an opportunity to ensure better code-reuse here.

Reply:
Not sure what you meant here, but I guess we can tell the same for other scripts in the directory. The only thing that was reused is the postgresql server startup wait logic